source/ocsp-support/tests/README.md

```diff
@@ -0,0 +1,182 @@
+could lead the driver or server to read stale data. See the
+[Appendix](../ocsp-support.md#os-level-ocsp-cache-manipulation) for instructions on how to clear OS-level OCSP caches.
+
+For each test, ensure that the mock OCSP responder is configured to use the correct certificate and to report the
+expected revocation status for that certificate (see
+[Mock OCSP Responder Testing Suite](#mock-ocsp-responder-testing-suite) for configuration details) and that a `mongod`
+is running with the correct certificate chain. The mock OCSP responder MUST BE started before the `mongod` as the
+`mongod` expects that a responder will be available upon startup. Again, each test MUST BE its own Evergreen task in
+order to minimize the impact of user-level and application-level OCSP caching
+
+To assert whether a test passes or fails, drivers SHOULD create a MongoClient with the options specified under "URI
+options", connect to a server and attempt to issue a ping command. The success or failure (due to a TLS error) of the
+ping command should correlate with the expected test result.
+
+Drivers may wish to use a smaller value for `serverSelectionTimeoutMS` to speed up tests (otherwise server selection
+will spin for the entire duration even after a driver encounters a TLS error early).
+
+### Testing on Linux
+
+Drivers MUST test on Linux platforms that have server support for OCSP stapling.
+
+- [SERVER-51364](https://jira.mongodb.org/browse/SERVER-51364) disables OCSP stapling on the server for Ubuntu 18.04.
+- [SERVER-56848](https://jira.mongodb.org/browse/SERVER-56848) is a known bug with Go clients and versions of RHEL 8
+  before 8.3.
+
+Consider using RHEL 7.0 or Ubuntu 20.04 as alternative platforms.
+
+### Testing on Windows and macOS
+
+Until [SPEC-1589](http://jira.mongodb.org/browse/SPEC-1589) and [SPEC-1645](https://jira.mongodb.org/browse/SPEC-1645)
+are resolved, drivers can only test with ECDSA certificates on Linux and thus, on Windows and macOS, drivers can only
+test with RSA certificates. Therefore, when testing on Windows and macOS, each column in the test matrix that utilizes
+an OCSP responder represents only two tests:
+
+1. A test with RSA certificates and an OCSP responder that uses the issuing CA's certificate
+2. A test with RSA certificates and an OCSP responder that uses a delegate certificate
+
+Additionally, because the Windows and macOS `mongod` do not support stapling when a client connects, the following sets
+of tests will be identical even if a driver supports stapled OCSP: {Test 1, Test 3} and {Test 2, Test 4}. Therefore,
+when testing on Windows and macOS, a driver MAY skip Test 1 and Test 2 if desired. A driver MAY also simply choose to
+run all the tests in the table, irrespective of OS, in order to simplify the testing procedure.
+
+## Changelog
+
+**2024-08-20**: Migrated from reStructuredText to Markdown.
+
+**2021-11-01**: Clarify that not all Linux platforms support server stapling.
```

Comment:
```suggestion
**2021-11-08**: Clarify that not all Linux platforms support server stapling.
```

To match commit date of https://github.com/mongodb/specifications/commit/3ccd0e5405a55a86623447ffe34fab3c8895823d. I forgot to update the date in https://github.com/mongodb/specifications/pull/986#discussion_r744875442.